mailauth/contrib/user/models.py

```diff
@@ -50,9 +53,24 @@ class AbstractEmailUser(AbstractUser):
     username = None
     password = None
 
-    email = CIEmailField(_("email address"), unique=True, db_index=True)
+    email = CIEmailField(
+        _("email address"), blank=True, null=True, unique=True, db_index=True
+    )
     """Unique and case insensitive to serve as a better username."""
 
+    email_hash = models.TextField(db_index=True, editable=False)
```

Comment:
The field will not be populated when updating the lib. Will it?

Reply:
We have a migration for the model that we ship. If anyone built a subclass, I will include an upgrade guide in the release notes.